nautobot/core/api/views.py

```diff
@@ -859,6 +861,49 @@ def get(self, request):
         return Response(object_counts)
 
 
+class GetSettingsView(NautobotAPIVersionMixin, APIView):
+    """
+    This view exposes Nautobot settings.
+
+    By default, it returns all serializable settings available in the system.
+    You can filter the settings by providing one or more `name` query parameters.
```

Comment:
This feels like a major security risk to me. `?name=SECRET_KEY`? `?name=NAPALM_PASSWORD`? Let's make this as narrowly-targeted of a feature as we possibly can rather than exposing every possible settings parameter.